### ISSUE TITLE
Broken tag crashes lore2sphinx

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5261]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5261|
|Type|defect|
|Created|2011-09-16 15:35:20Z|

As seen on the [buildbot](http://buildbot.twistedmatrix.com/builders/sphinx-documentation/builds/430/steps/shell_5/logs/stdio), a broken missing tag is screwing up the lore2sphinx conversion, this should fix it:

```
#!diff
Index: ../Twisted/trunk/doc/core/development/policy/svn-dev.xhtml
===================================================================
--- ../Twisted/trunk/doc/core/development/policy/svn-dev.xhtml	(revision 32533)
+++ ../Twisted/trunk/doc/core/development/policy/svn-dev.xhtml	(working copy)
@@ -212,7 +212,7 @@
 <h2>Emacs</h2>
 
 <p>A minor mode for development with Twisted using Emacs is available.  See 
-<code>twisted-dev.el</code>, provided by <a href="http://launchpad.net/twisted-emacs">,
+<code>twisted-dev.el</code>, provided by <a href="http://launchpad.net/twisted-emacs">twisted-emacs</a>,
 for several utility functions which make it easier to grep for methods, run test cases, etc.</p>
 
 <h2>Building Debian packages</h2>
```

Attachments:

* [svn.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/96e/96e4ffe10a111021957f4e07c7a30f0d60226a75/4a55897a0f97d26b008c7c5f3a5676ea02996f60.patch) (738 bytes) - added by thijs on 2011-09-16 17:45:13Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5261 5261
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1316187320000000 1316187320000000
changetime__1316276173000000 1316276173000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

lore2sphinx now exits properly when it fails, causing the buildslave to [turn red](http://buildbot.twistedmatrix.com/builders/sphinx-documentation/builds/431), where previously it would always be green.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Thanks, looks good (plus a misc news fragment), please apply.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [32538]) Apply svn.patch: Correct a broken link to the twisted-emacs project.

Author: thijs
Reviewer: exarkun
Fixes: [#5261](https://github.com/twisted/twisted/issues/5261)
